ste/xfer-anyToRemote.go

```diff
@@ -289,7 +290,24 @@ func epilogueWithCleanupSendToRemote(jptm IJobPartTransferMgr, s ISenderBase, si
 		}
 	}
 
-	s.Epilogue()
+	s.Epilogue() // Perform service-specific cleanup before jptm cleanup. Some services may actually require setup to make the file actually appear.
+
+	// In theory, due to the limits imposed in copy.go, this should never be true unless it's a S2S transfer.
```

Comment:
hmmm there is no such limit though

Reply:
Ah, yeah. I had a limit implemented a while back (https://github.com/Azure/azure-storage-azcopy/pull/478/commits/5902a7527a177033c27107b300bcf3c1218279db).

I'll cut out this comment.